docs/source-pytorch/fabric/advanced/collectives.rst

```diff
@@ -4,45 +4,237 @@
 Communication between distributed processes
 ###########################################
 
-Page is under construction.
+With Fabric you can easily access information about a process or send data between processes with a standardized API and agnostic to the distributed strategy.
+
 
 ----
 
 
-You can also easily use distributed collectives if required.
+*******************
+Rank and world size
+*******************
+
+The rank assigned to a process is a number in the range of *0, ..., world size - 1*, where *world size* is the total number of distributed processes.
+If you are using multi-GPU, think of the rank as the *GPU ID* or *GPU index*, although rank extends to distributed processing in general.
+
+The rank is unique across all processes, regardless of how they are distributed across machines, and it is therefore also called **global rank**.
+We can also identify processes by their **local rank**, which is only unique among processes runing on the same machine, but is not unique globally across all machines.
+Finally, each process is associated with a **node rank** in the range *0, ..., num nodes - 1*, which identifies on which machine (node) the process is running on.
+
+.. figure:: https://pl-flash-data.s3.amazonaws.com/fabric/docs/collectives/ranks.jpeg
+   :alt: The different type of process ranks: Local, global, node.
+   :width: 100%
+
+Here is how you launch multiple processes in Fabric:
 
 .. code-block:: python
 
-    fabric = Fabric()
+    from lightning.fabric import Fabric
 
-    # Transfer and concatenate tensors across processes
-    fabric.all_gather(...)
+    # Devices and num_nodes determine how many processes there are
+    fabric = Fabric(devices=2, num_nodes=3)
```

Comment:
If #14078 lands then we wouldn't want to mention num_nodes like this, no?

Reply:
Yes. In this case `world_size` would just be whatever is set externally on the cluster. For now it should be ok to show it like this, because it will take some work to complete https://github.com/Lightning-AI/lightning/issues/14078